### PR TITLE
chore: Added example file, updated .gitignore and README.

### PR DESCRIPTION
**Pull request by dhsrocha**
Commit for attending #49

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/java-http-client?pullRequest=54) <br/>All committers have signed the CLA.

**Comment by mbernier** [comment]
I merged in the master. The build should now pass

File: README.md [hunk 1]
```diff
@@ -136,6 +136,8 @@ cd examples
 javac -classpath {path_to}/sendgrid-java-http-client-4.0.0-jar.jar:. Example.java && java -classpath {path_to}/sendgrid-java-http-client-4.0.0-jar.jar:. Example
 ```
 
+You can use the ```.env_example``` file located at "examples" folder. Just copy it to a ```.env``` folder ou can put on your project's root.
```
Comment (mbernier) [inline README.md]: Can you please fix 
- `ou` to `you` ?

**Review by mbernier — DISMISSED** [review]

**Comment by thinkingserious** [comment]
Hello @dhsrocha,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/announcing-hacktoberfest-2017">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**